erpnext/healthcare/doctype/lab_test/lab_test.js

```diff
@@ -137,7 +137,7 @@ var get_lab_test_prescribed = function(frm){
 		});
 	}
 	else{
-		frappe.msgprint(__("Please select Patient to get Lab Tests"));
+		frappe.msgprint(__("Please select a Patient to get Lab Orders"));
```

Comment:
```suggestion
		frappe.msgprint(__("Please select a Patient to get Lab Prescriptions"));
```
Can we call it Lab Prescriptions just to be consistent with the child table naming in Patient Encounter?